From d745f1637f56ff2a3dd415df490bee972e062b03 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 04 Oct 2018 14:08:09 +0300
Subject: [PATCH] release: improve the jenkins job starting code

* Remove -q from parameters, so we get some output
* Move the URL to a variable for better editability
* Separate URL from build parameters
* Add jenkins build cause text

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 857e5e6..e3cf246 100644
--- a/Makefile
+++ b/Makefile
@@ -69,6 +69,10 @@
 
 RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader btl
 RELEASE_TAG_POINTER := latest-embedded-release
+RELEASE_JOB_URL := http://192.168.0.4:8080/job/goya-release-test/buildWithParameters
+RELEASE_JOB_CAUSE := "cause=Started_by_$(shell whoami)"
+RELEASE_JOB_TOKEN := "token=goya-release"
+RELEASE_JOB_PARAMS := $(RELEASE_JOB_TOKEN)&$(RELEASE_JOB_CAUSE)
 
 .PHONY: print-release-repos
 print-release-repos:
@@ -84,7 +88,7 @@
 	@-repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
-	@wget -q -O - "http://192.168.0.4:8080/job/goya-release-test/buildWithParameters?token=goya-release"
+	@wget -O - "$(RELEASE_JOB_URL)?$(RELEASE_JOB_PARAMS)"
 	@echo "Started the \"goya-release-test\" jenkins job... check your email for results."
 
 .PHONY: hl-goya hl-gaudi
